src/components/JobConfiguration.vue

```diff
@@ -204,9 +206,25 @@ export default defineComponent({
           }, {
             text: this.$t('Skip'),
             handler: () => {
-              if (job) {
-                this.store.dispatch('job/skipJob', job)
+              let skipTime = {};
```

Comment:
I think this logic should belong to action only
Also, we followed same in Job Manager so would be good for consistency
https://github.com/hotwax/job-manager/blob/main/src/store/modules/job/actions.ts#L562